addons/mail/static/src/component_hooks/use_drag_visible_dropzone/use_drag_visible_dropzone.js

```diff
@@ -31,12 +31,20 @@ function useDragVisibleDropZone() {
         document.addEventListener('dragenter', _onDragenterListener, true);
         document.addEventListener('dragleave', _onDragleaveListener, true);
         document.addEventListener('drop', _onDropListener, true);
+
+        // Thoses Events prevent the browser to open or download the file if
+        // it's dropped outside of the dropzone
+        window.addEventListener('dragover', (e) => e.preventDefault(), false);
+        window.addEventListener('drop', (e) => e.preventDefault(), false);
     });
 
     onWillUnmount(() => {
         document.removeEventListener('dragenter', _onDragenterListener, true);
         document.removeEventListener('dragleave', _onDragleaveListener, true);
         document.removeEventListener('drop', _onDropListener, true);
+
+        window.removeEventListener('dragover', (e) => e.preventDefault(), false);
+        window.removeEventListener('drop', (e) => e.preventDefault(), false);
```

Comment:
- no bracket with single param of arrow function
- for consistency, name `ev` instead of `e` (like with `catch`, we prefer `err` instead of `e`)
- 3rd param `false` is default behaviour when not provided, so not needed.